Description: ignore typescript ≥ 4.1 regression
Author: Xavier Guimard <yadd@debian.org>
Bug-Debian: https://bugs.debian.org/975794
Forwarded: no
Last-Update: 2021-10-27

--- a/src/promisify.ts
+++ b/src/promisify.ts
@@ -24,6 +24,7 @@
 					if (err) {
 						reject(err);
 					} else {
+						// @ts-ignore
 						resolve(rtn);
 					}
 				}
--- a/test/test.ts
+++ b/test/test.ts
@@ -250,6 +250,7 @@
 				await req({ agent, ...info });
 			} catch (err) {
 				gotError = true;
+// @ts-ignore
 				assert.equal(err.message, 'bad');
 			}
 
@@ -277,6 +278,7 @@
 				await req({ agent, ...info });
 			} catch (err) {
 				gotError = true;
+// @ts-ignore
 				assert.equal(err.message, 'bad');
 			}
 
